static_src/components/resource_usage.jsx

```diff
@@ -44,36 +44,31 @@ export default class ResourceUsage extends React.Component {
 
   render() {
     const props = this.props;
-    let title = <h2 className={ this.styler('stat-header')}>{ props.title }</h2>;
     let stat = (
       <Stat
         editable={ props.editable }
         max={ props.max }
         min={ props.min }
         onChange={ props.onChange }
+        title={ props.title }
         name={ props.name }
         primaryStat={ props.amountTotal }
-        secondaryInfo={ props.secondaryInfo }
+        secondaryInfo={ <span>{ props.secondaryInfo }</span> }
```

Comment:
Seems strange that the type for `secondaryInfo` on `ResourceUsage` is `string`, but for `Stat` it's `element`. We should change both to `node` which would be [any renderable type](https://facebook.github.io/react/docs/react-api.html#react.proptypes.node) including both strings and elements.